flickr_download/flick_download.py

```diff
@@ -368,16 +391,35 @@ def main():
                         help='Skip the actual download of the photo')
     parser.add_argument('-j', '--save_json', action='store_true',
                         help='Save photo info like description and tags, one .json file per photo')
+    parser.add_argument('-c', '--cache', type=str, metavar='CACHE',
+                        help='Cache results on provided file')
+    parser.add_argument("-v", "--verbose", action="store_true",
+                        help="increase output verbosity")
     parser.set_defaults(**_load_defaults())
 
     args = parser.parse_args()
 
+    if args.verbose:
+        logging.getLogger().setLevel(logging.DEBUG)
+
+    cache = None
+    if args.cache:
+        cache = get_cache(args.cache)
+        Flickr.enable_cache(cache)
+
+        def signal_handler(signal, frame):
+            save_cache(args.cache, cache)
+            sys.exit(0)
+        signal.signal(signal.SIGINT, signal_handler)
+
+        print('Cache is enabled')
+
     if args.list_naming:
         print(get_filename_handler_help())
         return 1
 
     if not args.api_key or not args.api_secret:
-        print ('You need to pass in both "api_key" and "api_secret" arguments', file=sys.stderr)
+        print('You need to pass in both "api_key" and "api_secret" arguments', file=sys.stderr)
```

Comment:
Why this change? It doesn’t follow the style of the rest of the file?

Reply:
pyflakes complained about this one, but I can revert

Reply:
Weird, I get no errors from `flake8` on it